From 4ee82c36e65fc51cc60c93386a04a6b817064e10 Mon Sep 17 00:00:00 2001
From: Mykola Lysenko <Mykola.Lysenko@amd.com>
Date: Thu, 17 Dec 2015 19:30:54 +0800
Subject: [PATCH 1197/1565] drm/amd/dal: attach tile MST connector property

As it is used in DRM code, it is better to be initialized

Change-Id: I9f42f18c9d5a519cda5e493ecebfac42b90c7249
Signed-off-by: Mykola Lysenko <Mykola.Lysenko@amd.com>
Acked-by: Jordan Lazare <Jordan.Lazare@amd.com>
---
 drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm_mst_types.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm_mst_types.c b/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm_mst_types.c
index 85fd17f..3750a0d 100644
--- a/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm_mst_types.c
+++ b/drivers/gpu/drm/amd/dal/amdgpu_dm/amdgpu_dm_mst_types.c
@@ -296,6 +296,10 @@ static struct drm_connector *dm_dp_add_mst_connector(struct drm_dp_mst_topology_
 		&connector->base,
 		dev->mode_config.path_property,
 		0);
+	drm_object_attach_property(
+		&connector->base,
+		dev->mode_config.tile_property,
+		0);
 
 	drm_mode_connector_set_path_property(connector, pathprop);
 
-- 
1.9.1

